src/style.css

```diff
@@ -1,3 +1,107 @@
-body {
+*,
+*::before,
+*::after {
+  padding: 0;
   margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+ul,
+ol,
+li {
+  list-style: none;
+}
+
+img {
+  vertical-align: top;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+html {
+  --blue: #00acdc;
+}
+
+body {
+  height: 100%;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header {
+  width: 100%;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  display: flex;
+  position: relative;
+}
+
+.nav__link {
+  height: 60px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding: 23px 0; instead